cmd/rosa-support/create/proxy/cmd.go

```diff
@@ -25,7 +25,7 @@ var Cmd = &cobra.Command{
 	Short: "Create proxy",
 	Long:  "Create proxy.",
 	Example: `  # Create a proxy
-  rosa-helper create proxy --region us-east-2 --vpc-id <vpc id>`,
+  rosa-support create proxy --region us-east-2 --vpc-id <vpc id> --zone <AZ> --ca-file <filepath> --keypair-name <name>`,
```

Comment:
NP: can we change the flag "zone" name to "--availability-zone", to be it more descriptive?